src/luarocks/cmd.lua

```diff
@@ -288,6 +286,102 @@ do
    end
 end
 
+local function get_status(status)
+   return status and "ok" or "not found"
+end
+
+local function get_config_text()
```

Comment:
Could you pass in `cfg` here? Gives us the opportunity in future to print/preview configs without setting globals